tests/unit/resolver/test_resolve_issue.py

```diff
@@ -141,9 +142,13 @@ def test_setup_sandbox_config_gitlab_ci_non_root(mock_getuid):
                 is_experimental=False,
             )
 
+<<<<<<< HEAD
+            assert_sandbox_config(openhands_config.sandbox, local_runtime_url='http://localhost', enable_auto_lint=True)
+=======
             assert_sandbox_config(
                 openhands_config.sandbox, local_runtime_url='http://localhost'
             )
+>>>>>>> 0c307ea12e9979a2958f674645b2e3c3b95dd295
 
```

Comment:
```suggestion
<<<<<<< HEAD
            assert_sandbox_config(openhands_config.sandbox, local_runtime_url='http://localhost', enable_auto_lint=True)
=======
            assert_sandbox_config(
                openhands_config.sandbox, local_runtime_url='http://localhost'
            )
>>>>>>> 0c307ea12e9979a2958f674645b2e3c3b95dd295

```

Looks like an unresolved merge conflict